packages/fhir-converter/src/templates/cda/DataType/Address.hbs

```diff
@@ -2,7 +2,7 @@
     "use": {{>ValueSet/AddressUse.hbs code=addr.use}},
     "line": [
        	{{#each (toArray addr.streetAddressLine)}}
-           	"{{{this._}}}",
+           	"{{this._}}",
```

Comment:
repro-ed bug here.

Reply:
I am going to leave this as double braces without the parseReferenceData given that the only place we have seen weird stuff with junky xml is with data getting pulled into ._ fields from the html content. 
So this {{{ here is just a vuln that can kill the xml